indicators/pollinatorIndicators/R/pollinatorIndicators.qmd

```diff
@@ -0,0 +1,1013 @@
+#| echo: false
+#| warning: false
+#| label: fig-asoDiversity
+#| fig-width: 9
+#| fig-height: 9
+
+
+g1 <- ggplot(asoDataMeadows) +
+  gg(asoDataMeadows["alpineReference"], aes(col = alpineReference)) +
+scale_color_gradientn(colours = terrain.colors(5, rev = TRUE), 
+                      breaks = seq(0, 1, 0.2), 
+                      limits = c(0, 1) )+
+  geom_sf(data = regionGeometry, alpha = 0.1)+
+  theme_bw()+
+  labs(color = " ",
+       title = "a) Alpine")
+
+g2 <- ggplot(asoDataMeadows) +
+  gg(asoDataMeadows["borealReference"], aes(col = borealReference)) +
+scale_color_gradientn(colours = terrain.colors(5, rev = TRUE), 
+                      breaks = seq(0, 1, 0.2), 
+                      limits = c(0, 1))+
+  geom_sf(data = regionGeometry, alpha = 0.1)+
+  theme_bw()+
+  labs(color = " ",
+       title = "b) Boreal")
+
+g3 <- ggplot(asoDataMeadows) +
+  gg(asoDataMeadows["temperateReference"], aes(col = temperateReference)) +
+scale_color_gradientn(colours = terrain.colors(5, rev = TRUE), 
+                      breaks = seq(0, 1, 0.2), 
+                      limits = c(0, 1))+
+  geom_sf(data = regionGeometry, alpha = 0.1)+
+  theme_bw()+
+  labs(color = " ",
+       title = "c) Temperate")
+
+g4 <- ggplot(asoDataMeadows) +
+  gg(asoDataMeadows["allReference"], aes(col = allReference)) +
+  scale_color_gradientn(colours = terrain.colors(5, rev = TRUE), 
+                      breaks = seq(0, 1, 0.2), 
+                      limits = c(0, 1))+
+  geom_sf(data = regionGeometry, alpha = 0.1)+
+  theme_bw()+
+  labs(color = " ",
+       title = "d) All zones")
+
+ggarrange(g1, g2, g3, g4,
+          ncol = 2, nrow = 2,
+          common.legend = TRUE,
+          legend = "bottom")
+```
+
+
+<!--# Describe the reference values used to set the lower and upper limits on the normative indicator scale. Why was the current option chosen and how were the reference values quantified? If the reference values are calculated as part of the analyses further down, please repeat the main information here. -->
+
+#### 4.2.2. Threshold value for defining *good ecological condition (if relevant)*
+
+We present three threshold values for defining a good condition. Any scales indicator greater than the scaled indicators (@fig-ecosystemCondition (a) are deemed to refer to `good conditions` at the location, otherwise `poor conditions`. We define three threshold values which ca be considered: 
+
+- Threshold of 0.5 [@fig-ecosystemCondition (b)]
+
+- Threshold of 0.6 [@fig-ecosystemCondition (c)]
+
+- Threshold of 0.75 [@fig-ecosystemCondition (d)]
+
+**What does this mean and how do we interpret these thresholds. Does these conditions make sense and how do we interpret those?**
+```{r}
+#| fig-cap: 'Shannon diversity indices at ASO data locations.'
+#| echo: false
+#| warning: false
+#| label: fig-ecosystemCondition
+#| fig-width: 9
+#| fig-height: 9
+  load("../data/ecosystemConditions.RData")
```

Comment:
This file does not exist.

Reply:
Look here: P:\412421_okologisk_tilstand_2024\pollinatorIndicators\oldScripts\Data\markdownResults\image